### PR TITLE
Configurations

### PR DESCRIPTION
**Pull request by DanielMwendwa**

### CODE REVIEW

File: code_schemes/ws_correct_dataset.json [hunk 1]
```diff
@@ -0,0 +1,1136 @@
...
+        {
+            "CodeID": "code-3194113f",
+            "CodeType": "Normal",
+            "DisplayText": "ssf dcf s01e04",
+            "StringValue": "ssf_dcf_s01e04",
+            "NumericValue": 102,
+            "VisibleInCoda": true,
+            "MatchValues": [
+                "ssf dcf s01e04"
+            ]
```
Comment (IsaackMwenda) [inline code_schemes/ws_correct_dataset.json:103]: we need to add rec and election ws code themes here.

File: configuration/pipeline_config.json [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        "3903ad8e-284e-48a0-881b-89c0b06b9334",
+        "45939459-e4d5-4ad6-8650-3e7134b9a035"
+      ]
+    },
+    {
+      "SourceType": "RapidPro",
+      "Domain": "textit.in",
+      "TokenFileURL": "gs://avf-credentials/SSF_DCF-SLD-Textit-Token.txt",
+      "ContactsFileName": "ssf_dcf_contacts",
+      "ActivationFlowNames": [
```
Comment (IsaackMwenda) [inline configuration/pipeline_config.json:25]: add activation flow for week 1 atleast

**Review by IsaackMwenda — APPROVED** [review]

File: configuration/pipeline_config.json (hunk 2 above)
Comment (DanielMwendwa) [reply configuration/pipeline_config.json:25]: Updated, thanks.

File: code_schemes/ws_correct_dataset.json (hunk 1 above)
Comment (DanielMwendwa) [reply code_schemes/ws_correct_dataset.json:224]: Updated, thanks.

**Pull request merged**